src/OpenTelemetry.Exporter.Console/OpenTelemetry.Exporter.Console.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <Authors>OpenTelemetry authors</Authors>
+    <PackageIconUrl>https://opentelemetry.io/img/logos/opentelemetry-icon-color.png</PackageIconUrl>
+    <PackageProjectUrl>https://OpenTelemetry.io</PackageProjectUrl>
+    <PackageTags>$(PackageTags);distributed-tracing;console</PackageTags>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OpenTelemetry\OpenTelemetry.csproj" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />

```

Comment:
nit: is it possible to use System.Text.Json and avoid non-platform dependencies?

Reply:
As far as I can see `System.Text.Json` requires .NET 4.6.1. This exporter could be changed to drop support for .NET 4.6, but it would be different from all the other providers. Should this be changed to .NET 4.6.1?